ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Superuser/tasks/main.yml

```diff
@@ -19,6 +20,23 @@
     - Superuser_Account == "Enabled"
   tags: [superuser, adoptopenjdk]
 
+- name: Create Superuser user's home folder (Solaris)
```

Comment:
Somewhat disappointing if this isn't the default

Reply:
unfortunately, the built-in ansible user module doesn't create the home folder so this was a workaround